.travis.yml

```diff
@@ -3,7 +3,8 @@
 language: node_js
 
 node_js:
-  - 6
+  - 10
+  - 12
```

Comment:
Does it now also make sense to add 14 to this list since that is in LTS?

tbh, PostCSS still has v10 support, so even though v10 is no longer supported, it's probably correct to leave it in this list

Reply:
Agreed, but I can fix this later.